cmake/Platform/Common/Configurations_common.cmake

```diff
@@ -75,3 +75,6 @@ set(CMAKE_POSITION_INDEPENDENT_CODE True)
 
 include(CheckPIESupported)
 check_pie_supported()
+
+# Determine if lld is installed to use as a default linker by supported platforms/configurations
+find_program(LLD_LINKER_INSTALLED lld)
```

Comment:
Good idea, I could add a check for an optional linker name variable and have that override lld if set, so the user can specify ld or any other linker. It would also give us a compiler-independent way of letting the user specify the linker since different compilers expect different flags & arguments

Reply:
Added a commit for this, we also need to update https://github.com/o3de/o3de.org/blob/main/content/docs/user-guide/build/reference.md if we decide to go this route

Reply:
I added a GHI to o3de.org for updating the documentation: https://github.com/o3de/o3de.org/issues/2424

Reply:
I'd suggest applying this to the gcc configurations as well